lib/archethic/p2p/node.ex

```diff
@@ -123,8 +86,31 @@ defmodule Archethic.P2P.Node do
           P2P.supported_transport(),
           reward_address :: binary(),
           origin_public_key :: Crypto.key(),
-          origin_key_certificate :: binary()
+          origin_key_certificate :: binary(),
+          mining_public_key :: Crypto.key()
         ) :: binary()
+  def encode_transaction_content(
+        {ip1, ip2, ip3, ip4},
+        port,
+        http_port,
+        transport,
+        reward_address,
+        origin_public_key,
+        key_certificate,
+        mining_public_key
+      ) do
+    <<encode_transaction_content(
+        {ip1, ip2, ip3, ip4},
+        port,
+        http_port,
+        transport,
+        reward_address,
+        origin_public_key,
+        key_certificate
+      )::binary, mining_public_key::binary>>
+  end
+
+  @doc false
```

Comment:
I don't get why you keep the old encode function since it is not used anymore I believe

Reply:
Indeed, at first see, it doesn't look relevant I will check, thx